api/v1alpha1/connection_types.go

```diff
@@ -9,20 +9,19 @@ import gwapiv1 "sigs.k8s.io/gateway-api/apis/v1"
 
 // Connection allows users to configure connection-level settings
 type Connection struct {
```

Comment:
will this be a little odd?
```yaml
connection:
  connectionLimit:
```

Reply:
its to tackle the issue outlined in #2805